### ISSUE TITLE
Strings: UTF-8 compatible `toLower`, `toUpper`, `downcase`, `upcase`

### ISSUE THREAD

**Issue by joshgoebel** [joshgoebel/wren-essentials#11]
Ref: https://github.com/sheredom/utf8.h

Lets add:

- [ ] `toLower`
- [ ] `toUpper`
- [ ] `downcase(_)` - now UTF-8 compatible
- [ ] `upcase(_)` - now UTF-8 compatible
- [ ] `capitalize` - I assume we'd bump this to (to build on the new utf8 stuff)
- [ ] `titlecase` - I assume we'd bump this to (to build on the new utf8 stuff)

Is there any reason to keep the existing upcase/downcase around as pure ASCII versions?

**Comment by joshgoebel** [comment]
@camilowser You want this one?

**Comment by clsource** [comment]
I don't see any reason to keep pure ASCII.

**Comment by clsource** [comment]
Upon testing the `wrenc` in REPL mode it cannot handle non ascii chars like `ñ` 

`Unhandled key-code [dec]: 195`

**Comment by joshgoebel** [comment]
Yeah, there is a PR for this that I should merge... but you can still test with scripts which work just fine.  It's only the REPL that can't handle it.
